container/src/services/container.service.ts

```diff
@@ -106,7 +106,11 @@ export class ContainerService {
                   {
                     msg: LuigiInternalMessageID.SEND_CONTEXT_HANDSHAKE,
                     context: targetCnt.context || {},
-                    internal: {},
+                    internal: {
+                      thirdPartyCookieCheck: {
+                        disabled: !!(targetCnt.getAttribute && targetCnt.getAttribute('skip-cookie-check') === 'true')

```

Comment:
What about : ` disabled: !!targetCnt.skipCookieCheck`. I am guessing that might work as well, and shorter.

Reply:
For some reason 'targetCnt.skipCookieCheck' always returns 'true' - not sure why